packages/ui/components/layout/Sidebar.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+import { Layout, Menu } from 'antd';
+import {
+  DashboardOutlined,
+  ProfileOutlined,
+} from '@ant-design/icons';
+import { APP_NAME } from 'constants/application';
+import { SIDEBAR_LINKS } from 'constants/lists';
+
+const ICONS = [<DashboardOutlined />, <ProfileOutlined />];
```

Comment:
Can we change `SIDEBAR_LINKS` to `[{ name: '', icon: <>}]`?

Reply:
we will remove antd, so will change accordingly :)